draw/src/main/java/chat/rocket/android/draw/widget/ColorSeekBar.kt

```diff
@@ -0,0 +1,181 @@
+package chat.rocket.android.draw.widget
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.Canvas
+import android.graphics.Shader
+import android.graphics.LinearGradient
+import android.graphics.Paint
+import android.graphics.RectF
+import android.util.AttributeSet
+import android.view.MotionEvent
+import android.view.View
+import androidx.annotation.ArrayRes
+import chat.rocket.android.draw.R
+
+class ColorSeekBar(context: Context, attributeSet: AttributeSet): View(context, attributeSet){
+
+    private val minThumbRadius = 16f
+    private var colorSeeds = intArrayOf(Color.parseColor("#000000"), Color.parseColor("#FF5252"), Color.parseColor("#FFEB3B"), Color.parseColor("#00C853"), Color.parseColor("#00B0FF"), Color.parseColor("#D500F9"), Color.parseColor("#8D6E63"))
```

Comment:
You can use our extension function `parseColor()`. Also you can also parse colors in this way:

`"black".parseColor()` - for "regular" colors.

Reply:
This extension function is in the `app` module.

Reply:
Try moving that extension to the `util` module then.
More important then using this existing extension is that it will prevent you from getting an exception since we're dealing with it (while you're not).